client/active-game/active-game-manager.js

```diff
@@ -117,10 +119,21 @@ export default class ActiveGameManager extends EventEmitter {
     if (current && current.config) {
       this.emit('gameCommand', gameId, 'routes', routes)
       this.emit('gameCommand', gameId, 'setupGame', current.config.setup)
-      this.emit('gameCommand', gameId, 'allowStart')
     }
   }
 
+  allowStart(gameId, waitTime = 2000) {
+    if (this.activeGame && this.activeGame.id !== gameId) {
+      return
+    }
+
+    this._setStatus(GAME_STATUS_STARTING)
+    setTimeout(() => {
+      this.emit('gameCommand', gameId, 'allowStart')
+      this.allowStartSent = true
```

Comment:
I never see this get set back to false anywhere, so it seems like once this happens once, this condition is no longer correct? I think it'd make more sense to put this condition inside the `this.activeGame`, that way we're guaranteed it gets erased if we're launching a new game

Reply:
Oops, you're right. Will fix.